openml-h2o/pom.xml

```diff
@@ -275,10 +275,19 @@
                             <transformers>
                                 <transformer
                                         implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
+                                <transformer
+                                        implementation="io.github.edwgiz.log4j.maven.plugins.shade.transformer.Log4j2PluginCacheFileTransformer" />
```

Comment:
Just asking. Should this go through a security review?

Reply:
I don't think that we need it. this doesn't even add any dependency to the released artifact. This is just required during package phase